docs/src/main/asciidoc/transaction.adoc

```diff
@@ -188,14 +188,14 @@ The default value is 60 seconds.
 
 == Configuring transaction node name identifier
 
-Narayana, as the underlaying transaction manager, has an concept of unique node identifier.
-This is important if you consider to use XA transactions with involve multiple resources.
+Narayana, as the underlying transaction manager, has a concept of unique node identifier.
```

Comment:
It probably should be `has a concept of a unique` (added `a`) if we are going to be really strict. I'll leave it up to you to decide